daemon/dns/dns_resolvconf_file.go

```diff
@@ -94,3 +113,93 @@ func restoreDNS() error {
 	}
 	return internal.FileWrite(resolvconfFilePath, out, internal.PermUserRWGroupROthersR)
 }
+
+func tryToRestoreDNS() {
+	// if target is symlink, probably it is managed by other software - do nothing
+	if internal.IsSymLink(resolvconfFilePath) {
```

Comment:
it might be good to have the same login for VPN disconnect :smile:

Reply:
yes, implemented recovery options on regular `restoreDNS`